cpmpy/solvers/utils.py

```diff
@@ -72,6 +73,7 @@ def base_solvers():
                 ("pysat", CPM_pysat),
                 ("pysdd", CPM_pysdd),
                 ("minizinc", CPM_minizinc),
+                ("rc2", CPM_RC2)
```

Comment:
Just a thought: maybe we can group the list of solvers for each paradigm, or sort them alphabetically?